assemblies/assembly-enabling-authentication.adoc

```diff
@@ -35,4 +35,13 @@ For each provider that you want to use, follow the dedicated procedure to comple
 . Configure {product} to use the authentication provider.
 
 include::proc-adding-azure-as-an-authentication-provider.adoc[leveloffset=+1]
-include::modules/auth/proc-adding-gitlab-oauth-as-an-authentication-provider.adoc[leveloffset=+1]
+
+
+include::assembly-auth-provider-github.adoc[leveloffset=+1]
+
+
+include::assembly-auth-provider-oidc.adoc[leveloffset=+1]
+
+
+//include::modules/auth/proc-adding-gitlab-oauth-as-an-authentication-provider.adoc[leveloffset=+1]
```

Comment:
Can we also add the GitHub and OIDC auth providers sections from the Getting Started guide to this assembly?

Reply:
The associated Jira (RHIDP-3016) calls for the following actions: 

- Replace the Enabling the GitLab OAuth authentication module with Enabling the GitHub OAuth authentication provider
- Add an Enabling Keycloak with OIDC authentication module to the assembly (currently in the Getting Started guide) 

It looks like this change resolves the issue.